app/directives/cityValidator.js

```diff
@@ -0,0 +1,37 @@
+// Validator directive to ensure only cities in `{City}, {State}` format are accepted.
+(function () {
+    angular
+        .module('weatherApp')
+        .directive('cityValidator', directiveFunction);
+
+    function directiveFunction () {
+        return {
+            restrict: 'A',
+            require: 'ngModel',
+            link: linkFunction
+        };
+
+        function linkFunction(scope, elm, attrs, ctrl) {
+            ctrl.$validators.potato = function(modelValue) {
+                console.log(modelValue);
+
+                var re1='((?:[a-z][a-z]+))';  // Word 1
```

Comment:
(Note: I assume this is provided from some tutorial. It's not exactly bulletproof, granted it's an exercise and probably meant to be simple; it only needs to contain this pattern, so something like `"San Jose, CA"` passes because of `"Jose, CA"`...`"((*@#*(&Jose, CA&*&**&#"` also passes.)